frontend/packages/operator-lifecycle-manager/src/components/operand/create-operand.tsx

```diff
@@ -58,15 +59,9 @@ export const CreateOperand: React.FC<CreateOperandProps> = ({
   const { data: csv } = clusterServiceVersion;
   const { data: crd } = customResourceDefinition;
   const [helpText, setHelpText] = React.useState(FORM_HELP_TEXT);
-  const next =
-    activePerspective === 'dev'
-      ? '/topology'
-      : `${resourcePathFromModel(
-          ClusterServiceVersionModel,
-          match.params.appName,
-          match.params.ns,
-        )}/${match.params.plural}`;
```

Comment:
I don't think we can change the location we go to after creating an operand. The user who was in the `installed operators` nav item will now be redirected to the workloads tab even though an operand may not result in a workload being created.

Reply:
Removed this change